gulpfile.js

```diff
@@ -373,42 +376,51 @@ if (this.IS_NODE_JS) {
   module.exports = Blockly;
 } else {
   document.write('<script src="' + this.BLOCKLY_DIR +
-      '/closure/goog/base.js"></script>');
+      '/${closurePath}/base.js"></script>');
   document.write('<script>this.BLOCKLY_BOOT(this);</script>');
 }
 `;
 
 let deps = [];
-return gulp.src('core/**/**/*.js')
+const srcs = ['core/**/**/*.js'];
```

Comment:
Can you factor any of this out into its own method since we do the same thing in compressed?